src/System.IO.FileSystem.AccessControl/src/System.IO.FileSystem.AccessControl.builds

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.IO.FileSystem.AccessControl.csproj">
```

Comment:
You don't need the empty TargetGroup metadata you can make the entry. <Project Include="System.IO.FileSystem.AccessControl.csproj" />